### DISCUSSION TITLE
Flesh out test plan

### DISCUSSION THREAD

**Discussion by inexorabletash** [a2stuff/a2d#728] [General]
The list of tests to run in `res/notes/testplan.txt` should be fleshed out with the most of fixes and additions made so far. The doc should be something that we can have volunteers run through on a variety of hardware/emulator setups to make sure the eventual 1.2 release doesn't make anything worse.

**Comment by inexorabletash** [comment]
@c0mmander8 maybe you want to take this on?

1. Clone the repo to your own account
2. Edit your copy of the file (which can be done on GitHub)
3. When you have a set of changes ready for review (not more often than once a day), submit a Pull Request

This would be very helpful and would benefit from your attention to detail.

**Comment by inexorabletash** [comment]
@c0mmander8 - here's an example of what I mean: 

4ec682dbdcbb779ebaef43cdb216d602d5b27964

I'm slowly going through all the closed bugs and seeing what should be documented as tests to run before identifying a release. This could use additions - it's not complete! - and organization (e.g. consistent format, structure, etc).

This is much more valuable than release note edits!

**Comment by inexorabletash** [comment]
@I3arana - maybe you could contribute here?

First task is to go back through old resolved bugs and describe the repro steps concisely here. 

Eventually I'd like to get this edited into a checklist we could ask people to follow to verify correct behavior on various hardware and localization configurations.

**Comment by I3arana** [comment]
Thats not a half.bad.idea

**Comment by inexorabletash** [comment]
@c0mmander8 - I dug through the history and issues 1-206 predate the inclusion of the test plan document. It would be helpful for you to look through that range of issues, identify any that could benefit from a test description to prevent regressions, and suggest additions to the document here.

**Comment by inexorabletash** [comment]
1d1d907947b07eb49ca413dca29c04efb1fa010f adds cases for older bug fixes.
